CONTRIBUTING.md

```diff
@@ -28,9 +28,14 @@ is merged to `master`.
 **Please squash and merge** so that the commit history on master looks cleaner and easier to navigate.
 
 ### Testing
-Please run/modify the [tests](tests) each time a change is made to the logic or structure. You can run tests using:
+Please run/modify the [tests](tests) each time a change is made to the logic or structure. Make sure you have a local `mongo` instance running or else all tests will fail. You can run the tests using:
 ```shell
-pytest -v tests/ --doctest-modules
+py.test-3 -v tests/api --doctest-modules
```

Comment:
Can we keep the command as `pytest` instead of `py.test-3`? I think the specific command depends on how the tools are installed (for me in my python3 virtualenv `pytest -v tests/api --doctest-modules` works fine).